cypress-tests/cypress/e2e/PaymentTest/00004-NoThreeDSAutoCapture.cy.js

```diff
@@ -96,4 +96,52 @@ describe("Card - NoThreeDS payment flow test", () => {
       cy.retrievePaymentCallTest(globalState, data);
     });
   });
+
+  context("Card-NoThreeDS payment with shipping cost", () => {
+    let shouldContinue = true; // variable that will be used to skip tests if a previous test fails
+
+    beforeEach(function () {
+      if (!shouldContinue) {
+        this.skip();
+      }
+    });
+
+    it("create-payment-call-test", () => {
+      const data = getConnectorDetails(globalState.get("connectorId"))[
+        "card_pm"
+      ]["PaymentIntentWithShippingCost"];
```

Comment:
Have this been tested with other connectors? Asking so because, by default, `501` will be thrown for connectors that do not have this config.

Reply:
Yes , tested with other connectors also